### ISSUE TITLE
ncspot not working

### ISSUE THREAD

**Issue by YorvTwmRepo** [hrkfdn/ncspot#1226]
Before i format my pc, ncspot worked correctly, no errors, but now it dosent work, and idk how to fix, already downloaded all dependencies btw.

![Captura de tela de 2023-07-11 11-20-10](https://github.com/hrkfdn/ncspot/assets/132029754/3965fa05-c484-4760-abe5-029721077925)

OS: Void Linux XFCE
Terminal: XFCE Terminal
Version: 1.13.3
Installed from: Official Void Linux repo

**Comment by aragonnetje6** [comment]
are you on IPv6? i am as of recently, and get a similar issue

**Comment by mdgjohnny** [comment]
same issue here - anybody knows how to fix this?

**Comment by psaumur** [comment]
> same issue here - anybody knows how to fix this?

Nope. Same issue here in 2026. Tried suggested changes to my /etc/hosts file, VPNs, several different methods of installation and so far - nothing.
